src/hotspot/cpu/aarch64/register_aarch64.hpp

```diff
@@ -29,6 +29,11 @@
 #include "asm/register.hpp"
 #include "utilities/powerOfTwo.hpp"
 
+#ifdef _WIN32
+#undef INTERNAL_VISIBILITY
+#define INTERNAL_VISIBILITY
+#endif
```

Comment:
Don't make this change.  This sort of conditionalization belongs with the definition of INTERNAL_VISIBILITY.
Similarly for register_riscv, register_x86, and vmreg.